test/functional/helpers.py

```diff
@@ -0,0 +1,142 @@
+# taken from https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/gaia_test.py#L858
+# XXX factor out into utility object for use by other tests
+from marionette_driver.by import By
+from marionette_driver.errors import NoSuchElementException, StaleElementException
+# noinspection PyUnresolvedReferences
+from marionette_driver import Wait
+
+import pyperclip
+import re
+import urlparse
+import time
+
+from serversetup import ROOMS_WEB_APP_URL_BASE
+from config import USE_LOCAL_STANDALONE
+
+
+def wait_for_element_displayed(self, by, locator, timeout=None):
```

Comment:
I think what might be slightly nicer, is to change the functions in this file to be within a class. The class. Something like:

```python
class LoopTestDriver():
    def setUp(self, marionette):
        self.marionette = marionette
```

Then in `test_*.py` we should be able to do:

```python
class Test1BrowserCall(LoopTestDriver, MarionetteTestCase):
    def setUp():
        # start server
        self.loop_test_servers = LoopTestServers()

        MarionetteTestCase.setUp(self)
        LoopTestDriver.setUp(self, marionette)

        ...
```

I think that will work, if not https://github.com/mozilla/talkilla/tree/master/test/functional may also help. It is not quite the same - as we don't have web driver, but the ideas should be similar.